app/views/components/link.scala.html

```diff
@@ -15,6 +15,5 @@
  *@
 
 @import models.ui.Link
-
 @(link: Link)
-<a class="govuk-link" href="@link.dest" @if(link.window) {target="_blank"}>@link.text</a>
\ No newline at end of file
+<a @if(!link.asButton) {class="govuk-link"} href="@link.dest" @if(link.asButton) {class="govuk-button" role="button"} @if(link.window) {target="_blank"}>@link.text</a>
```

Comment:
I think you also need 'data-module="govuk-button"' for accessibility. Have a look at the examples from manage-external-guidance-frontend.

Reply:
added and pushed